data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,353 @@
+			`	They give you access to the ship and proceed to explain the details. "It is of no real significance, but surely you understand we cannot go trough official means for this, we will have to present it as a fine. Worry not, we will make it something of no consequence, like an old Korath forbidden tech you would not have known about, that we had to confiscate."`
+			`	You quickly realize they are not asking you for your opinion at all. In fact, you notice the fine has already been issued before you could say anything. "Oh, and you should not change its colors, you'll understand why later."`
+				accept
+
+mission "Hai Legacy 2"
+	shipyard
+	landing
+	name "Young but not that young"
+	description "Use the newly acquired old Ladybug to bring aid to the Unfettered Hai."
+	clearance
+	deadline 30
+	cargo "food (aid)" 121
+	destination "Firelode"
+	blocked "You need space for the agreed rush cargo of food to Unfettered space on the Ancient Ladybug, and should use it as flagship."
+	to offer
+		has "ship model: Ancient Ladybug"
+	on enter
+		dialog `As you take of from the planet you notice a message was left on the bridge, probably from the "associates". "You should use the Ladybug as your flagship and you should probably not bring any other ships. You'll understand why later."`
+			`	Given the dangers of Unfettered space, you figure it would be a good idea to familiarize yourself with the design, and maybe to modify it.`
+	on enter
+		system
+			attributes "unfettered"
+		"old reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)"
+		"reputation: Hai (Unfettered)" >?= 0
+		dialog `As you enter the system, you prepare for a fight to get the cargo trough, but, surprisingly, it is very quiet this time.`
+			`	Indeed, for the first time, entering Unfettered space does not mean getting shot at, potentially by missiles all sides.`
+			`	They are not hailing you either. Perhaps they think you are one of the Hai aiding them, especially given your cargo and the ship you are flying.`
+	on fail
+		"reputation: Hai (Unfettered)" <?= -1000
+		dialog `The Unfettered and their associates are not clearly happy when you tell them about this. "You will have to prove yourself another way."`
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		dialog `You receive a message from the 'associates of Hai-home', "thank you for helping our brethren, <last>. We will not forget this. Please accept these <payment> to get you started on paying back."`
+		payment 100000
+		log "Delivered a pressing food cargo to the Unfettered using a newly acquired but pretty old Ladybug. They seem to not be firing on me, for now at least."
+
+
+
+# no category, meaning you can't remove it from the Ladybug
+outfit "Unfettered Landing Permit"
+	"maintenance costs" 681
+	description "This item is needed in order to land on Unfettered planets, and its loss will cancel the arrangement."
```

Comment:
What value does the description have? You don't see it.

(If you wanted the license to be visible you could add a variable like `shiplocked` or something you you can't remove it but can see it.)

Reply:
I see. (Or rather, I don't because it's only readable if you look at this file!) Thanks for the explanation!